src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,11 +1,6 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
Wildcard imports violate the coding standard. Please use single imports and change your settings in Intellij: `Settings` > `Editor` > `Code Style` > `Java` > `Imports` and set both class count and names count to 99 (or any large number)